### DISCUSSION TITLE
Free tier Dockerhub for teams is disappearing

### DISCUSSION THREAD

**Discussion by rocky** [Mathics3/mathics-omnibus#15] [General]
I recently got this email from "Docker" no-reply@docker.com: 

> Free Team organizations are a legacy subscription tier that no longer exists. This tier included many of the same features, rates, and functionality as a paid Docker Team subscription.
>
> After reviewing the list of accounts that are members of legacy Free Team organizations, we’ve identified yours as potentially being one of them.
>
> If you own a legacy Free Team organization, access to paid features — including private repositories — will be suspended on April 14, 2023 (11:59 pm UTC). Upgrade your subscription before April 14, 2023 to continue accessing your organization.
> 
> If you don’t upgrade to a paid subscription, Docker will retain your organization data for 30 days, after which it will be subject to deletion. During that time, you will maintain access to any images in your public repositories, though rate limitations will apply. At any point during the 30-day period, you can restore access to your organization account if you upgrade to a paid subscription. Visit our [FAQ](https://em.docker.com/NzkwLVNTQi0zNzUAAAGKf9wl3r0J5T0edk5XIR3NamREoR6AU9JcP9Z0Go0ivA4kx9I9JVALdYeZIj1aX6_sjcw752skxwEqxLA=) for more information.

Some information about our specific use. We don't have any private repositories. The rate of downloads is not likely to run into any rate limitation. Lastly our team currently just has one person, me.

So with this here are some options: 

* Pay dockerhub $300/year
* Move things under my individual account. 
* Switch to [GitLab](https://docs.gitlab.com/ee/user/packages/container_registry/)
* Something else?

**Comment by mmatera** [comment]
Gitlab sounds good, but I do not know how much work it would require.
